helper/test/table.c

```diff
@@ -32,10 +32,10 @@ int main(int argc ODP_UNUSED, char *argv[] ODP_UNUSED)
 	char ip_addr1[] = "12345678";
 	char ip_addr2[] = "11223344";
 	char ip_addr3[] = "55667788";
-	char mac_addr1[] = "0A1122334401";
-	char mac_addr2[] = "0A1122334402";
-	char mac_addr3[] = "0B4433221101";
-	char mac_addr4[] = "0B4433221102";
+	char mac_addr1[] = { 30, 31, 32, 33, 34, 35, };
```

Comment:
Now the values are no longer null terminated printable strings but there is still printing code that assumes they are.